quickstart/101-front-door-classic/readme.md

```diff
@@ -0,0 +1,131 @@
+# Azure Front Door (classic)
+
+This template deploys an [Azure Front Door (classic)](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/frontdoor).
+
+## Resources
+
+| Terraform Resource Type | Description |
+| - | - |
+| `azurerm_resource_group` | The resource group for all the deployed resources.|
+| `azurerm_frontdoor` | The Front Door (classic). |
+
```

Comment:
Please include the random_id resource here as well.

Reply:
Updated.